compiler/src/middle_end/analyze_inline_wasm.re

```diff
@@ -59,7 +59,7 @@ let analyze = ({imports, body, analyses}) => {
   mod_has_inlineable_wasm := false;
   let process_import = ({imp_use_id, imp_desc}) => {
     switch (imp_desc) {
-    | GrainValue("runtime/unsafe/memory", name) =>
+    | GrainValue("runtime/unsafe/memory.gr", name) =>
```

Comment:
Wouldn't this break the optimization for any unsafe code that doesn't exist within the stdlib / wouldn't we need to check for both to handle library code too?

Reply:
I set it up so that the `.gr` extension would be injected automatically on the filepath so it would be applied, but you bringing this up is actually making me see that this logic is a little busted if you're using relative imports. For example if you're in the `runtime/unsafe` directory then doing `include "./memory.gr"` wouldn't trigger this optimization

Reply:
https://github.com/grain-lang/grain/issues/1990